snakemake_executor_plugin_slurm/__init__.py

```diff
@@ -31,13 +32,51 @@
 
 @dataclass
 class ExecutorSettings(ExecutorSettingsBase):
+    logdir: Optional[str] = field(
+        default=f"/home/{os.environ['USER']}/.snakemake/slurm_logs",
+        metadata={
+            "help": """
+                   Per default the SLURM log directory (writing output is
+                   required by SLURM) is '~/.snakemake/slurm_logs'.
+                   This flag allows to set an alternative directory.
+                   """,
+            "env_var": False,
+            "required": False,
+        },
+    )
```

Comment:
done. This is actually safer because `/home/$USER` might not exist on a cluster. This code was experimental, now relying on the `~`. Is committed. Will be pushed.